docs/install-manual.md

```diff
@@ -198,45 +197,49 @@ rules to allow inbound traffic on the required port numbers.)
 
-1. To create the primary manager in a high-availability Swarm cluster, use the following syntax:
+1. Use SSH to connect to the `manager0` instance and use `ifconfig` to get its IP address.
 
-        $ docker run -d -p 4000:4000 swarm manage -H :4000 --replication --advertise <manager0_ip>:4000 consul://<consul_ip>:8500
+        $ ifconfig
 
-    Because this is particular manager is on the same `manager0` and `consul0`
-    instance as the consul node, replace both `<manager0_ip>` and `<consul_ip>`
-    with the same IP address. For example:
+2. To create the primary manager in a high-availability Swarm cluster, use the following syntax:
 
-        $ docker run -d -p 4000:4000 swarm manage -H :4000 --replication --advertise 172.30.0.161:4000 consul://172.30.0.161:8500
+        $ docker run -d -p 4000:4000 swarm manage -H :4000 --replication --advertise <manager0_ip>:4000 consul://<consul0_ip>:8500
+
+    Replacing `<manager0_ip>` and `<consul0_ip>` with the IP address from the previous command, for example:
 
-2. Enter `docker ps`.
+        $ docker run -d -p 4000:4000 swarm manage -H :4000 --replication --advertise 172.30.0.125:4000 consul://172.30.0.161:8500
 
-    From the output, verify that both a Swarm cluster and a consul container are running.
-    Then, disconnect from the `manager0` and `consul0` instance.
+3. Enter `docker ps`.
 
-3. Connect to the `manager1` node and use `ifconfig` to get its IP address.
+    From the output, verify that a Swarm cluster container is running.
+    Then, disconnect from the `manager0` instance.
+
+4. Connect to the `manager1` node and use `ifconfig` to get its IP address.
 
         $ ifconfig
 
-4. Start the secondary Swarm manager using following command.
+5. Start the secondary Swarm manager using following command.
 
       Replacing `<manager1_ip>` with the IP address from the previous command, for example:
 
         $ docker run -d -p 4000:4000 swarm manage -H :4000 --replication --advertise <manager1_ip>:4000 consul://172.30.0.161:8500
 
-5. Enter `docker ps`to verify that a Swarm container is running.
+6. Enter `docker ps`to verify that a Swarm container is running. Then disconnect from the `manager1` instance.
 
-6. Connect to `node0` and `node1` in turn and join them to the cluster.
+7. Connect to `node0` and `node1` in turn and join them to the cluster.
 
     a. Get the node IP addresses with the `ifconfig` command.
 
     b. Start a Swarm container each using the following syntax:
 
-        docker run -d swarm join --advertise=<node_ip>:2375 consul://<consul_ip>:8500
+        docker run -d swarm join --advertise=<node_ip>:2375 consul://<consul0_ip>:8500
 
       For example:
 
         $ docker run -d swarm join --advertise=172.30.0.69:2375 consul://172.30.0.161:8500
 
+    c. Enter `docker ps` to verify that a Swarm container is running.
```

Comment:
I am afraid you can not check by `docker ps`, because in https://github.com/docker/swarm/blob/master/docs/swarm-api.md, it says:

```
GET "/containers/json"	Containers started from the swarm official image are hidden by default, use all=1 to display them.
```